packages/discovery-react-components/src/components/DocumentPreview/components/HtmlView/HtmlView.tsx

```diff
@@ -40,6 +40,9 @@ const SANITIZE_CONFIG = {
     // bbox
     'page'
   ],
+  FORBID_TAGS: ['input', 'form', 'a', 'button', 'script'],
```

Comment:
Could you add a link to [this](https://github.com/cure53/DOMPurify#can-i-configure-dompurify) (or other documentation if there's a better version somewhere) right above this constant declaration?